me.id.webverify/webverifylib/src/main/res/layout/activity_web_view.xml

```diff
@@ -1,15 +1,25 @@
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-                xmlns:tools="http://schemas.android.com/tools"
-                android:layout_width="match_parent"
-                android:layout_height="match_parent"
-                tools:context=".WebViewActivity"
-                >
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    tools:context=".WebViewActivity"
+    >
+
+    <android.support.v7.widget.Toolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        app:titleTextColor="#2fc073"
+        android:background="#ff2e3d50"
+        android:layout_height="wrap_content"/>
 
     <WebView
-        android:layout_width="fill_parent"
-        android:layout_height="fill_parent"
+        android:layout_width="match_parent"
+        android:layout_below="@+id/toolbar"
```

Comment:
is this line in the appropriate order? `layout_width`, then `layout_height` then alphabetically is the rule i think